x/project/internal/types/params.go

```diff
@@ -59,10 +60,18 @@ func (p Params) String() string {
 `, p.ProjectMinimumInitialFunding, p.IxoDid)
 }
 
+func validateAccessConfig(i interface{}) error {
+	ok := i
+	if ok != nil {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+	return sdkerrors.Wrap(ErrInternal, "unknown type")
+}
+
 // Implements params.ParamSet
 func (p *Params) ParamSetPairs() params.ParamSetPairs {
 	return params.ParamSetPairs{
-		{KeyIxoDid, &p.IxoDid},
-		{KeyProjectMinimumInitialFunding, &p.ProjectMinimumInitialFunding},
+		{KeyIxoDid, &p.IxoDid, validateAccessConfig},
+		{KeyProjectMinimumInitialFunding, &p.ProjectMinimumInitialFunding, validateAccessConfig},
```

Comment:
Similar to payments module, we need one validation function per parameter.

Reply:
please give an example for one would be easy for me